### PR TITLE
FIX: Lightning wallet CSV export does not differentiate between paid …

### PR DESCRIPTION
**Pull request by marcosrdz**
…and expired invoices #5835

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ahw735wxglohvaoqjxi6bih5l4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11920)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: screen/wallets/details.js [hunk 1]
```diff
@@ -417,30 +417,44 @@ const WalletDetails = () => {
...
           const bb = Buffer.from(hash);
           hash = bb.toString('hex');
         }
       }
-      csvFile += '\n' + [new Date(transaction.received).toString(), hash, value, memo].join(','); // CSV line
+
+      const data = [new Date(transaction.received).toString(), hash, value, memo];
+
+      if (wallet.chain === Chain.OFFCHAIN) {
+        // Replace 'condition' with your specific condition
```
Comment (Overtorment) [inline screen/wallets/details.js:453]: whats this comment?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: screen/wallets/details.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/details.js]: My bad

**Pull request merged**